wp-content/plugins/AB-Content-Alterer/ab-content-alterer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/*
+Plugin Name:  AB Content Alterer
+Description:  WordPress Content Alterer: Change Unbounce and Leadpages in content to <strong>Instapage</strong>
+Version:      0.7
+Author:       andrzej.bialowas@instapage.com
+*/
+defined( 'ABSPATH' ) or die( 'No script kiddies please!' );
+add_action('the_content', 'abAlterContent');
+function abAlterContent($abTextToAlter) {
+  $abTextToAlter = strtolower($abTextToAlter);

```

Comment:
You are modifying `$abTextToAlter` so passing `I like leadpages` would result in `i like Instapage` which is incorrect. Use `str_ireplace` without `strtolower instead`